src/NUnitFramework/framework/Internal/ParamAttributeTypeConversions.cs

```diff
@@ -86,8 +88,17 @@ public static object Convert(object value, Type targetType)
         }
 
         /// <summary>
-        /// Converts a single value to the <paramref name="targetType"/>, if it is supported.
+        /// Performs several special conversions allowed by NUnit in order to
+        /// permit arguments with types that cannot be used in the constructor
+        /// of an Attribute such as TestCaseAttribute or to simplify their use.
         /// </summary>
+        /// <param name="value">The value to be converted</param>
+        /// <param name="targetType">The target <see cref="Type"/> in which the <paramref name="value"/> should be converted</param>
+        /// <param name="convertedValue">If conversion was successfully applied, the <paramref name="value"/> converted into <paramref name="targetType"/></param>
+        /// <returns>
+        /// <c>true</c> if <paramref name="value"/> was converted and <paramref name="convertedValue"/> should be used;
+        /// <c>false</c> is no conversion was applied and <paramref name="convertedValue"/> should be ignored
+        /// </returns>
```

Comment:
This xmldoc comment was copied over from `TestCaseAttribute`'s "PerformSpecialConversion"
https://github.com/nunit/nunit/pull/3098/files#diff-012b7fc50160d70b26586addebf7ce1fL395